src/seedu/addressbook/data/person/Email.java

```diff
@@ -49,6 +49,10 @@ public boolean equals(Object other) {
                 && this.value.equals(((Email) other).value)); // state check
     }
 
+    @Override
+    public String getPrintableString()  {
```

Comment:
Remember to pay attention to the coding standard provided =).
There are 2 ` ` between the method signature `getPrintableString()` and `{`!